### DISCUSSION TITLE
can only add beatmap to 1 collection at a time?

### DISCUSSION THREAD

**Discussion by ampersand64** [ppy/osu#29589] [Ideas] [answered]
Adding a beatmap to a collection causes the collections submenu to disappear. 
Adding a single map to multiple collections is a common use case. 
It makes more sense to keep the submenu open, and just toggle the beatmap's inclusion in each collection.

**Comment by Joehuu** [comment, marked as answer]
Right-clicking the menu item will not close the submenu.
